app/views/hyrax/base/show.json.jbuilder

```diff
@@ -1,2 +1,12 @@
-json.extract! @curation_concern, *[:id] + @curation_concern.class.fields.reject { |f| [:has_model].include? f }
+j = json.extract! @curation_concern, *[:id] + @curation_concern.class.fields.reject { |f| [:has_model].include? f }
 json.version @curation_concern.etag
+
+if @curation_concern.is_a? Valkyrie::Resource
```

Comment:
should we push this logic into a service or helper somewhere?

it seems like it might be useful to avoid having this code live in the view, especially since it seems likely to churn a bit as the details get sorted out.

what do you think is best?

Reply:
nevermind, i think this is fine in the jbuilder file